x11-misc/rofi-wayland/rofi-wayland-1.7.5.ebuild

```
@@ -0,0 +1,63 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+EAPI=8
+inherit meson xdg
+
+DESCRIPTION="A window switcher, run dialog and dmenu replacement"
+HOMEPAGE="https://github.com/lbonn/rofi"
+SRC_URI="https://github.com/lbonn/rofi/releases/download/${PV}%2Bwayland1/rofi-${PV}+wayland1.tar.xz"
+S=${WORKDIR}/rofi-${PV}+wayland1
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+drun test +windowmode"
+RESTRICT="!test? ( test )"
+
+BDEPEND="
+	sys-devel/bison
+	sys-devel/flex
+	virtual/pkgconfig
+"
+RDEPEND="
+	dev-libs/glib:2
+	x11-libs/cairo[X,xcb(+)]
+	x11-libs/gdk-pixbuf:2
+	x11-libs/pango[X]
+	x11-libs/startup-notification
+	x11-libs/xcb-util
+	x11-libs/xcb-util-cursor
+	x11-libs/xcb-util-wm
+	x11-misc/xkeyboard-config
+	dev-libs/wayland
+	dev-libs/wayland-protocols
+"
+DEPEND="
+	${RDEPEND}
+	x11-base/xorg-proto
+	test? ( >=dev-libs/check-0.11 )
+"
+
+src_configure() {
+	local emesonargs=(
```

We usually put all the options that a package offers into the ebuild, even though they have the standard values, so meson_options.txt is this:
```
option('drun', type: 'boolean', value: true, description: 'Desktop file mode')
option('window', type: 'boolean', value: true, description: 'Window switcher mode')
option('check', type: 'feature', description: 'Build and run libcheck-based tests')
option('wayland', type: 'feature', value: 'enabled', description: 'Build with wayland support')
option('xcb', type: 'feature', value: 'enabled', description: 'Build with X11/xcb support')
```
which should become this I think:
```suggestion
	local emesonargs=(
	$(meson_use drun)
	$(meson_use windowmode window)
	$(meson_feature test check)
	-Dwayland=enabled
	-Dxkb=disabled
```

/ edit: I still don't know how this suggestion feature works in github, but I think you get the idea.

---

@thesamesam  I am not quite sure if this is correct with the test flag, though.